chapter-05/layered/money/convert_test.go

```diff
@@ -0,0 +1,51 @@
+package money_test
+
+import (
+	"testing"
+
+	"github.com/ablqk/tiny-go-projects/chapter-05/layered/money"
+)
+
+func TestConvert(t *testing.T) {
+	tt := map[string]struct {
+		amount          string
+		from            string
+		to              string
+		targetPrecision int
+		rateRepo        stubRate
+		validate        func(t *testing.T, got string, err error)
+	}{
+		"34.98 USD to EUR": {
+			amount:          "34.98",
+			from:            "USD",
+			to:              "EUR",
+			targetPrecision: 2,
+			rateRepo:        stubRate{rate: 1.2564},
+			validate: func(t *testing.T, got string, err error) {
```

Comment:
I am not a big fan of this validate function honestly I find it less readable...